lib/collectionspace/tools/xml.rb

```diff
@@ -107,17 +96,24 @@ def self.add_group_list(
       )
         return unless elements.any?
 
-#         puts "\n\nKEY: #{key}"
-#         puts "ELEMENTS:"
-#         pp(elements)
-#         puts "SUBKEY: #{sub_key}"
-#         puts "SUBELEMENTS:"
-#         pp(sub_elements)
+        #puts "\nELEMENTS FOR KEY: #{key}:"
+        #pp(elements)
+        #         puts "SUBKEY: #{sub_key}"
+        #         puts "SUBELEMENTS:"
+        #         pp(sub_elements)
 
         xml.send("#{key}#{list_suffix}".to_sym) {
           elements.each_with_index do |element, index|
             xml.send("#{key}#{group_suffix}".to_sym) {
-              element.each {|k, v| xml.send(k.to_sym, v)}
+              element.each {|k, v|
+                if v.is_a?(String)
+                  xml.send(k.to_sym, v)
+                elsif v.is_a?(Hash)
+                  xml.send(k.to_sym) {
+                    v.each{ |hk, hv| xml.send(hk.to_sym, hv) }
+                    }
+                end
```

Comment:
@Paulmulonzia @mark-cooper -- adding this logic loop allows us to treat structured date fields like regular fields within a field group list. They don't need to be handled as another nested list/group level, so they can be repeatable.